.github/workflows/build.yml

```diff
@@ -0,0 +1,150 @@
+name: Build
+
+on:
+  push:
+    branches: main
```

Comment:
Looks like CI is not triggered on this PR?

Wondering if this is because it comes from a fork. If so we could push to a branch on the repo directly if needed.

Reply:
Yes, I remembered the same thing append when creating a workflow in a PR.
I can reopen this PR from a branch on the repo, to trigger the tests (which I haven't modified yet, so at least the integration would have failed).

Reply:
Yes and then we should be able to continue from forks.